webui/react/src/components/ThemeToggle.module.scss

```diff
@@ -30,22 +30,16 @@ $transtion-type: ease-in-out 0.6s;
 }
 .system {
   background-color: gold;
-  border-bottom-color: gold;
-  border-left-color: black;
+  border-color: black gold gold black;
```

Comment:
👍🏼 did it automatically do this?

Reply:
I changed it by hand but `--fix` option is available

Reply:
that reminds me in saas I needed to add stylelink as part of `make fmt` same as we did to eslint (it looks like I haven't landed the commit though). this might be a good time to add it here as well. (or we can leave it for another pr)

Reply:
Even though I dont think adding it into `make fmt` would cause any issues, probably another PR would be good.

Reply:
yeah ofc